web_widget_image_download/static/src/xml/web_widget_image_download.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Flavio Corpa <flavio.corpa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<template>
+    <t t-extend="FieldBinaryImage">
+        <t t-jquery=".oe_form_binary_file_edit" t-operation="after">
+            <a class="fa fa-download fa-1g col-md-4 oe_form_binary_file_download" title="Download" download="avatar.png"></a>
```

Comment:
@Yajo its the [standard HTML5 download attribute](http://www.w3schools.com/tags/att_a_download.asp), required to download the image.